sdk/RELEASE.md

```diff
@@ -10,6 +10,7 @@
 ## Deprecations
 
 ## Bug Fixes and Other Changes
+* fix(sdk): bump JsonSchema pinned version to next major version. Fixes [#8648](https://github.com/kubeflow/pipelines/issues/8648) [#8855](https://github.com/kubeflow/pipelines/pull/8855)
```

Comment:
Sorry -- missed this the first time: could you change this to `chore(sdk)` in the release note and the PR title?

Can you also add the specific version change? E.g., "bump JsonSchema pinned version to >=3.0.1,<5"?